fuseops/ops.go

```diff
@@ -23,6 +23,12 @@ import (
 // File system
 ////////////////////////////////////////////////////////////////////////
 
+// OpContext contains context of the file system operation.
```

Comment:
Maybe `OpContext contains extra context that may be needed by some filesystems. See https://libfuse.github.io/doxygen/structfuse__context.html`